inc/CollectionFiller.hh

```diff
@@ -27,11 +27,14 @@ namespace mu2e{
           using Comment=fhicl::Comment;
           fhicl::Atom<int> diagLevel{Name("diagLevel"), Comment("for info"),0};
           fhicl::Atom<art::InputTag>chTag{Name("ComboHitCollection"),Comment("chTag"), "makePH"};
+	  fhicl::Atom<art::InputTag>tcTag{Name("TimeClusterCollection"),Comment("ttcTag"), "makePH"};
           fhicl::Atom<art::InputTag>cluTag{Name("CaloClusterCollection"),Comment("cluTag")};
           fhicl::Sequence<art::InputTag>kalSeedTag{Name("KalSeedCollection"),Comment("kalseedTag")};
           fhicl::Atom<art::InputTag>cosmicTrackSeedTag{Name("CosmicTrackSeedCollection"),Comment("cosmicTrackSeedTag")};
-          fhicl::Sequence<art::InputTag>MCTrajTag{Name("MCTrajectoryCollection"),Comment("MCTrajTag")};
```

Comment:
Is there a reason you switched?

Reply:
Yes, I was comparing it with the TEve code and its given as fhicl::Atom and not Sequence for the MCTrajectoryCollection. Since, there was an error with the MC truth helix display I was editing the code to be similar to the one in TEve.  

I have also made some other additions/changes like :

![Screenshot 2022-01-05 at 15 44 33](https://user-images.githubusercontent.com/83946783/148236896-66e59601-4dce-4832-b277-bc766de82eed.png)

![Screenshot 2022-01-05 at 15 51 57](https://user-images.githubusercontent.com/83946783/148237786-58684f8a-43a2-4aa3-ba82-52df9a94afbf.png)

So, now there is no Segmentation Fault ( which used to be present earlier) when I was trying to run the display, even when "AddMCTraj" is set to false. But, we need to look into this more to get the MC truth display up and running.

Reply:
Yes, I am going to work on this next. I am trying to change everything into Sequences eventually so we can have multiple collections plotted. It's fine for now but once this is merged I suggest you pull the latest code and work from that. I have opened another PR which I will merge after yours you can start from comparing your work to what's in there and then add you corrections on top. I've not tested it yet at all so it will probably still seg fault, I will look into that in the coming days once both our PR's are merged in. You are welcome to change anything I've done.